### PR TITLE
Refactor/#156 organize packages

### PR DESCRIPTION
**Pull request by ductiletoaster**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -211,38 +204,38 @@ workflows:
       - lint-packages:
           requires:
             - compile-packages
-      - test-packages:
-          requires:
-            - compile-packages
-      - compile-examples:
+# EXITING BECAUSE OF MEM????
```
Comment (jimmynotjim) [inline .circleci/config.yml]: Still need this?

File: packages/pixeloven-core/common/tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "extends": "../../tsconfig",
+    "extends": "../../../tsconfig",
```
Comment (jimmynotjim) [inline packages/pixeloven-core/common/tsconfig.json]: Any way to add a custom path to avoid having to change this if the organization changes again?

Comment (ductiletoaster) [reply packages/pixeloven-core/common/tsconfig.json]: Not that I can find. Eventually I want to overhaul the whole config setup but this is a built in function of TS itself

File: .circleci/config.yml (hunk 1 above)
Comment (ductiletoaster) [reply .circleci/config.yml]: I redid this again so it should be fixed. Going to try and resolve the CircleCI issues tomorrow.

**Pull request merged**